inst/templates/widget_js_new.txt

```diff
@@ -0,0 +1,19 @@
+HTMLWidgets.widget({
+
+  name: '%s',
+
+  type: 'output',
+
+  factory: function(el, width, height) {
+
+    return {
```

Comment:
Might be nice to add some spacing around the function as we had in the previous template